src/commands/install.yml

```diff
@@ -12,6 +12,10 @@ parameters:
       but before verifying Cypress and saving cache.
     type: string
     default: ""
+  install-browsers:
+    type: boolean
+    default: false
+    description: Flag to install additional browsers to run your tests. This is only needed if you are passing the `--browser`` flag in your `cypress-command`. This includes Chrome, FireFox, ChromeDriver and GeckoDriver. If you need additional browser support you can set this to false and make sure you use an executor with a docker image that includes the browsers of your choosing see https://hub.docker.com/r/cypress/browsers/tags
```

Comment:
Same thing here with GeckoDriver ref.

We might want to call out what the deafult is when not pasisng this flag as well.